container/sagemaker/python_service.py

```diff
@@ -168,6 +167,17 @@ def on_post(self, req, res):
                 'error': str(e)
             }).encode(encodings.utf_8.getregentry().name)  # pylint: disable=E1101
 
+    def on_delete(self, req, res, model_name):  # pylint: disable=W0613
+        try:
+            msg = self.grpc_client.delete_model(model_name)
+            res.body = msg
+            res.status = falcon.HTTP_200
+        except Exception as e:  # pylint: disable=W0703
+            res.status = falcon.HTTP_400
+            res.body = json.dumps({
+                'error': str(e)
+            }).encode(encodings.utf_8.getregentry().name)  # pylint: disable=E1101
```

Comment:
this is not very pythonic. it should just be 'utf-8' string (i see this is in many locations now)

Reply:
Updated.